### PR TITLE
People update

### PR DESCRIPTION
**Pull request by mosharaf**

### CODE REVIEW

File: source/people/index.md [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
 * [Yiwen Zhang](https://web.eecs.umich.edu/~yiwenzhg/)
 * Sanjay S. Singapuram
 * [Jiachen Liu](http://www-personal.umich.edu/~amberljc/)
+* [Jae-Won Chung](https://jaewonchung.me/)
+* [Insu Jang](https://insujang.github.io/)
 
 ## Current Master’s and Undergraduate Students
-* [Xiangfeng Zhu](https://xzhu27.me/) (CS-LSA BS)
 * [Yinwei Dai](https://dywsjtu.github.io/) (CSE MSE)
+* Chuheng Hu (CS BS @JHU)
```
Comment (Romero027) [inline source/people/index.md:24]: Chuheng has a personal website: https://chuheng001.github.io/

File: source/people/index.md [hunk 2]
```diff
@@ -38,6 +45,7 @@ date: 2020-07-22 23:09:46
 * Yuchen Wang (CSE BSE) → Quantcast
 * Jingyuan Zhu (CSE BSE) → PhD @Michigan
 * Alexander Neben (CSE BSE) → Ocient
+* Xiangfeng Zhu (CS-LSA BS) → PhD @ UWashington
```
Comment (Romero027) [inline source/people/index.md:48]: is the whitespace intended?

**Review by Romero027 — APPROVED** [review]
lgtm. Congrats again Dr. Gu! @juncgu

**Comment by mosharaf** [comment]
Updated. Thanks Allen

**Pull request merged**